samples/protected-setting-sample/README.md

```diff
@@ -0,0 +1,33 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Protected settings sample
+
+This sample shows how ISV can create protected settings and input secret values in UIDefinition as well as consume them in ARM Template so that they can be passed to Kubernetes Extension Type as Configuration Protected Settings.
```

Comment:
Add explanation on what protected settings are. Refer docs links below. Some details...

Protected configuration settings help manage sensitive data securely. These settings typically include information such as passwords, API keys, certificates, or connection strings.
Protected configuration settings are stored as secrets on the Kubernetes clusters. As a result, the key names of the protected configuration settings should be of simple text and special chars should not be included.
Protected configuration settings for an extension instance are stored for up to 48 hours in the cluster extension services. As a result, if the cluster remains disconnected during the 48 hours after the extension resource is created in Azure, the extension changes from a Pending state to a Failed state. To prevent this, we recommend that you bring clusters online regularly.

Reply:
https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-extensions
https://learn.microsoft.com/en-us/azure/aks/deploy-extensions-az-cli